composer.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "email-type-prediction/email-type-prediction",
+    "description": "lib for email type prediction",
+    "type": "library",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Yozhef",
+            "email": "hisemjo@gmail.com"
+        },
+        {
+            "name": "Yonchi",
+            "email": "2yonchi@gmail.com"
+        }
+    ],
+    "require": {
+        "php": "^7.4"
```

We need support PHP 8.0 
```suggestion
        "php": ">=7.4"
```